tests/integration/test.replication.js

```diff
@@ -839,7 +900,9 @@ adapters.forEach(function (adapters) {
             return db2.changes({style: 'all_docs'});
           }).then(function (res2) {
             changes2 = simplifyChanges(res2);
-
+            if (testUtils.isSyncGateway() && adapters[0] === 'http') {
+              return true;
+            }
```

Comment:
The reason I'm skipping HTTP in this case is because there's stuff worth investigating behind that flag. Thanks for reminding me.